### DISCUSSION TITLE
Clickable URL in note

### DISCUSSION THREAD

**Discussion by JonesyAtHome** [iSoron/uhabits#1634] [Feature Requests]
I would like to add a clickable URL to the note field so that I can e.g. open a website with workout tips directly from the habit.

**Comment by vnagel** [comment]
+1

The current workaround is to edit the habit to be able to highlight/copy the link, which is inconvenient and not conducive to performing the habit if visiting the link is a requirement (e.g. following along to an exercise video online).

![Screenshot_2023-03-31-17-44-01-68_83e306ed2b1f229c43c2a5339388c52e](https://user-images.githubusercontent.com/33532015/229238908-f55bb13d-a873-46a9-8849-c70fb4a8436e.jpg)
